doc/classes/Object.xml

```diff
@@ -510,88 +610,118 @@
 				node.SetIndexed("position", new Vector2(42, 0));
 				node.SetIndexed("position:y", -10);
-				GD.Print(node.Position); // (42, -10)
+				GD.Print(node.Position); // Prints (42, -10)
 				[/csharp]
 				[/codeblocks]
+				[b]Note:[/b] In C#, [param property] must be in snake_case when referring to built-in Godot properties.
 			</description>
 		</method>
 		<method name="set_message_translation">
 			<return type="void" />
 			<param index="0" name="enable" type="bool" />
 			<description>
-				Defines whether the object can translate strings (with calls to [method tr]). Enabled by default.
+				If set to [code]true[/code], allows the object to translate messages with [method tr] and [method tr_n]. Enabled by default. See also [method can_translate_messages]. 
 			</description>
 		</method>
 		<method name="set_meta">
 			<return type="void" />
 			<param index="0" name="name" type="StringName" />
 			<param index="1" name="value" type="Variant" />
 			<description>
-				Adds, changes or removes a given entry in the object's metadata. Metadata are serialized and can take any [Variant] value.
-				To remove a given entry from the object's metadata, use [method remove_meta]. Metadata is also removed if its value is set to [code]null[/code]. This means you can also use [code]set_meta("name", null)[/code] to remove metadata for [code]"name"[/code]. See also [method has_meta] and [method get_meta].
-				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the inspector and should not be edited.
+				Adds or changes the entry [param name] inside the object's metadata. The metadata [param value] can be any [Variant], although some types cannot be serialised correctly.
+				If [param value] is [code]null[/code], the entry is removed. This is the equivalent of using [method remove_meta]. See also [method has_meta] and [method get_meta].
+				[b]Note:[/b] Metadata that has a [param name] starting with an underscore ([code]_[/code]) is considered editor-only. Editor-only metadata is not displayed in the Inspector dock and should not be edited.
 			</description>
 		</method>
 		<method name="set_script">
 			<return type="void" />
 			<param index="0" name="script" type="Variant" />
 			<description>
-				Assigns a script to the object. Each object can have a single script assigned to it, which are used to extend its functionality.
-				If the object already had a script, the previous script instance will be freed and its variables and state will be lost. The new script's [method _init] method will be called.
+				Attaches [param script] to the object, and calls the script's [method _init] to instantiate it. A [Script] is used to extend the object's functionality.
+				If a script already exists, its instance is detached and its property values and state are lost, unless it's the same as [param script]. Built-in property values are still kept.
```

Comment:
Same problem as `_init()` description.

This method does not call the script's `_init()` to instantiate it. It instantiates the script and attaches the script instance. The `_init()` call is the result of instantiating the script.

> unless it's the same as [param script]

Skipping for unchanged value is a common behavior of setters I think.

Reply:
> Skipping for unchanged value is a common behavior of setters I think.

It is, indeed, but I thought that the effects of changing a script were more... "catastrophic" than most setters, so it needed a mention. It can (?) be removed since some user that needs this information can probably notice it with its own eyes...??